ci/render_pipelines_and_trigger_job

```diff
@@ -111,18 +107,22 @@ else
   create_credentials
 fi
 
-ci/render_task.py \
-  --outfile "${rendered_task}" \
+ci/render_all.py \
+  --pipeline_cfg "${pipeline_cfg}" \
+  --flavour-set "${FLAVOUR_SET}" \
+  --outfile-pipeline-main "${outfile_pipeline_main}" \
+  --outfile-pipeline-packages "${outfile_pipeline_packages}" \
+  --outfile-tasks "${rendered_task}" \
   ${RENDER_TASK_EXTRA_ARGS}
 
 # XXX hardcode other resources for now
 
 for manifest in \
   "${rendered_task}" \
-  "${outfile}" \
-  "${outfile_packages}"
+  "${outfile_pipeline_main}" \
+  "${outfile_pipeline_packages}"
 do
-  echo "Skip apply of ${manifest}"
+  echo "Apply of ${manifest}"
```

Comment:
was the original echo wrong semantically? 🤔

Reply:
yes, typo